### ISSUE TITLE
Change license to a permissive one

### ISSUE THREAD

**Issue by emcyaman** [SU-HPC/sparsebase#116]
MIT or Apache might be good options to make it permissive.

**Comment by AmroAlJundi** [comment]
You're right. We've discussed it internally and we're gonna change the license to MIT. Thank you for the suggestion.
